setup/paths.sh

```diff
@@ -37,38 +37,27 @@ else
     source $BUILD_HOME/cmsgemos/setup/paths.sh
 fi
```

Comment:
Yes okay, I understand.  However, when requesting changes in the future it would be more useful to understand the "why" that is attached to a request.  I think this would provide a greater understanding and allow developers to have a higher change of doing things correctly "the first time"